app.js

```diff
@@ -1 +1,7 @@
-'use strict';
+// 'use strict';
```

Comment:
Please don't submit corpse code, even if you plan on using it later. 

One thing you can do is just not include it as a script tag yet in your HTML if it's something that has no bearing yet on anything else.